client/src/components/util/Checkbox.tsx

```diff
@@ -0,0 +1,20 @@
+import { Img, Icon } from '@chakra-ui/react';
+import { JSXElementConstructor, ReactElement } from 'react';
+
+export default function Checkbox(props: any): ReactElement<any, string | JSXElementConstructor<any>> {
+  const { isIndeterminate, isChecked, ...reset } = props;
+  console.log(props);
+  return (
+    <>
+      {isChecked ? (
+        <Icon viewBox='0 0 19 19' {...reset}>
+          <Img src='images/checkbox_true_1.svg' />
+        </Icon>
+      ) : (
+        <Icon viewBox='0 0 19 19' {...reset}>
+          <Img src='images/checkbox_false_1.svg' />
+        </Icon>
+      )}
+    </>
+  );
+}
```

Comment:
```suggestion
  return (
      <Icon viewBox='0 0 19 19' {...reset}>
          <Img src={isChecked ? 'images/checkbox_true_1.svg' : ''images/checkbox_false_1.svg''} />
      </Icon>
  );
}
```

이런식으로 안한 이유가 있는지 궁금합니다!

Reply:
체크박스 커스텀하다가 실패한 코드라 이 파일은 삭제를 하도록 하겠습니다